c2corg_ui/static/js/feed.js

```diff
@@ -150,41 +402,107 @@ app.FeedController.prototype.handleFeed = function(response) {
   var data = response['data']['feed'];
   var token = response['data']['pagination_token'];
   this.nextToken = token;
-  for (var i = 0; i < data.length; i++) {
-    this.documents.push(data[i]);
+
+  this.initDocumentsCol_();
+  if (window.innerWidth < 1400) {
```

Comment:
Looks like there's a lot of code duplicated from ``feedColumnManager()`` here. Couldn't factorize a bit or use ``feedColumnManager()``?

Reply:
Maybe a little, but the two function dont really do the same thing. The feed ColumnManager resort all the column depending of the screen size. It's more simple to maintain that with separate functions.